.github/actions/common_setup/action.yml

```diff
@@ -26,6 +26,16 @@ runs:
             echo "The GITHUB_JOB_OVERRIDDEN ENV is unset, and must be set for the nested jobs"
             exit 1
           fi
+    - name: Setup zram
+      shell: bash
+      run: |
+          sudo modprobe zram
```

Comment:
Please add a comment explaining the intention of this code block, something like:
to make sure some of the tests with heavy RAM usage actually pass, we allocate 2% of RAM as  SWAP on a ZRAM.